LocalSettings.php

```diff
@@ -3746,6 +3746,11 @@
 			"*",
 		],
 	],
+	
+	// ScratchBlocks
+	'wgScratchBlocks4BlockVersion' => [
+		'default' => '3.0',
```

Comment:
This should be int format or MWS type changed to text and default changed to string.

Reply:
Done.